rviz_marker_tools/src/marker_creation.cpp

```diff
@@ -276,6 +276,20 @@ vm::Marker& makeMesh(vm::Marker& m, const std::string& filename, double sx, doub
 	return m;
 }
 
+vm::Marker& makeArrow(vm::Marker& m, const Eigen::Vector3d& start_point, const Eigen::Vector3d& end_point,
+                      double diameter, double head_length) {
+	// scale.y is set according to default proportions in rviz/default_plugin/markers/arrow_marker.cpp#L61
+	// for the default head_length=0, the head length will keep the default proportion defined in arrow_marker.cpp#L106
+	m.scale.x = diameter;
+	m.scale.y = 2 * diameter;
+	m.scale.z = head_length;
+	prepareMarker(m, vm::Marker::ARROW);
+	m.points.resize(2);
+	tf::pointEigenToMsg(start_point, m.points[0]);
+	tf::pointEigenToMsg(end_point, m.points[1]);
```

Comment:
It would have been better to directly use tf2 functions. See https://github.com/ros-planning/moveit_task_constructor/issues/78#issuecomment-826788870. But I guess, Michael will give you this as a next  task anyway :wink: